src/Infrastructure/LeanCode.Firebase.FCM/EntityFramework/PushNotificationTokenStore.cs

```diff
@@ -32,13 +34,31 @@ public async Task<List<string>> GetTokensAsync(Guid userId, CancellationToken ca
             return res.AsList();
         }
 
+        public async Task<Dictionary<Guid, List<string>>> GetTokensAsync(List<Guid> userIds, CancellationToken cancellationToken = default)
```

Comment:
`List<Guid> userIds` → `IReadOnlySet<Guid> userIds` or `IReadOnlyCollection<Guid> userIds` maybe?

Reply:
`IReadOnlySet` is a better option, thanks!